README.md

```diff
@@ -58,7 +58,10 @@ If you have chosen the first option, you can follow the theme reference instruct
 Following the point #2 you will need to upload your theme using theme installer.
 
 - Ensure that your root directory does not contain any .git hidden folders or files
+- Follow the base theme folder directory even when the folder are empty
+- Make sure the root folder is named `theme`
 - Compress it to a zip file
+- Name your zip file `theme`
```

Comment:
I was able to import with a ZIP not names theme. Only line 62 is true regarding this.